database/src/main/postgres/runs/V1.8.3__get_partitioning_checkpoints.sql

```diff
@@ -28,53 +28,64 @@ CREATE OR REPLACE FUNCTION runs.get_partitioning_checkpoints(
     OUT measured_columns           TEXT[],
     OUT measurement_value          JSONB,
     OUT checkpoint_start_time      TIMESTAMP WITH TIME ZONE,
-    OUT checkpoint_end_time        TIMESTAMP WITH TIME ZONE
+    OUT checkpoint_end_time        TIMESTAMP WITH TIME ZONE,
+    OUT has_more                   BOOLEAN
 )
-    RETURNS SETOF record AS
-$$
-    -------------------------------------------------------------------------------
+RETURNS SETOF record AS
+-------------------------------------------------------------------------------
 --
--- Function: runs.get_partitioning_checkpoints(JSONB, INT, TEXT)
---      Retrieves all checkpoints (measures and their measurement details) related to a
+-- Function: runs.get_partitioning_checkpoints(4)
+--      Retrieves checkpoints (measures and their measurement details) related to a
```

Comment:
I would explicitly document here in the fnc desc that one record being returned does not equal to a checkpoint, i.e. something that puts emphasis on fact that `i_limit` = X doesn't mean that no more than `X` records will be returned - it can be actually multiples of that, those pagination-related parameters & mechanism is all about checkpoints, not # of data returned, which might confuse someone

Reply:
ok added an explanation that the limit is related to number of checkpoints, not to amount of returned rows and that usually there is more than one row per checkpoint